arcgis-ios-sdk-samples/Augmented reality/View hidden infrastructure in AR/README.metadata.json

```diff
@@ -0,0 +1,43 @@
+{
+    "category": "Augmented reality",
+    "description": "Visualize hidden infrastructure in its real-world location using augmented reality.",
+    "ignore": false,
+    "images": [
+        "view-hidden-infrastructure-in-AR-1.png",
+        "view-hidden-infrastructure-in-AR-2.png"
+    ],
+    "keywords": [
+        "augmented reality",
+        "full-scale",
+        "infrastructure",
+        "lines",
+        "mixed reality",
+        "pipes",
+        "real-scale",
+        "underground",
+        "visualization",
+        "visualize",
+        "world-scale",
+        "AGSGraphicsOverlay",
+        "AGSMultilayerPolylineSymbol",
+        "AGSSketchEditor",
+        "AGSSolidStrokeSymbolLayer",
+        "AGSSurface",
+        "ArcGISARView"
+    ],
+    "redirect_from": [],
+    "relevant_apis": [
+        "AGSGraphicsOverlay",
+        "AGSMultilayerPolylineSymbol",
+        "AGSSketchEditor",
+        "AGSSolidStrokeSymbolLayer",
+        "AGSSurface",
+        "ArcGISARView"
+    ],
+    "snippets": [
+        "ViewHiddenInfrastructureARPipePlacingViewController.swift",
+        "ViewHiddenInfrastructureARViewController.swift",
```

Comment:
Should this view controller go first? Usually the initial view controller of a sample is named `[Some variation of the sample name]ViewController`. If this is not the initial view controller, perhaps this should be the name of the other one?

Reply:
I was really hesitant on picking a proper name for the second vc 🤦 . Seemingly it should be named as `ViewHiddenInfrastructureARViewerViewController`, but the 3 "view" in its name are too confusing...

So I'll change it to `ViewHiddenInfrastructureARExplorerViewController`.